data/tlg0059/tlg018/tlg0059.tlg018.perseus-eng2.xml

```diff
@@ -35,20 +35,23 @@
 		<note>Perseus 2.0</note>
 	</notesStmt>
 <sourceDesc>
-<biblStruct>
-	<monogr>
-		<author>Plato</author>
-		<title>Plato in Twelve Volumes, translated by W.R.M. Lamb.</title>
-		
-		<imprint>
+	<biblStruct>
+		<monogr>
+			<author>Plato</author>
+			<title>Plato in Twelve Volumes</title>
+			<editor role="translator">Walter Rangeley Maitland Lamb</editor>
+			<imprint>
+				<pubPlace>Cambridge, MA</pubPlace>
+				<publisher>Harvard University Press</publisher>
+				<pubPlace>London</pubPlace>
+				<publisher>William Heinemann Ltd.</publisher>
+				<date>1927</date>
```

Comment:
for dates post 1922 we have attribute type="printing" here (to correspond with the notation in the cts files

Reply:
hi @lcerrato yes I commented on the other pull request that I'm going to do a final pass through these files to standardize all the dates in CTS and the structured bibliographic data with "printing"